python/ray/util/state/common.py

```diff
@@ -743,6 +743,8 @@ class TaskState(StateSchema):
     task_log_info: Optional[dict] = state_column(detail=True, filterable=False)
     #: Task error detail info.
     error_message: Optional[str] = state_column(detail=True, filterable=False)
+    # Is task paused by the debugger
+    is_debugger_paused: Optional[bool] = state_column(detail=True, filterable=True)
```

Comment:
nit: i wonder if we can add more metadata about the debugging? e.g. do we know where this task is stopped, at which port or etc, other than just a boolean? Would that be possible?

Reply:
The only other part that we need is the port and that is already set on the worker, for now the behavior we need is to list tasks filtered by this flag, and only after the user select the task we can then get the worker to get the ip, and the debugger port. So we really do have everything we need here.